script/release/uploaders/upload.py

```diff
@@ -68,7 +68,7 @@ def main():
   upload_electron(release, electron_zip, args)
   if get_target_arch() != 'mips64el':
     symbols_zip = os.path.join(OUT_DIR, SYMBOLS_NAME)
-    shutil.copy2(os.path.join(OUT_DIR, 'symbols.zip'), symbols_zip)
+    shutil.copy2(os.path.join(OUT_DIR, 'breakpad_symbols.zip'), symbols_zip)
```

Comment:
Where does this file actually get created?

Reply:
oops, i started down a path of getting gn to make the zip file too but then half-undid it. i'll revert this to `symbols.zip`, which is what `zip-symbols.py` creates.